cmd/channel/email/main.go

```diff
@@ -38,17 +44,49 @@ func (ch *Email) SendNotification(req *plugin.NotificationRequest) error {
 	}
 
 	var msg bytes.Buffer
-	_, _ = fmt.Fprintf(&msg, "To: %s\n", strings.Join(to, ","))
-	_, _ = fmt.Fprintf(&msg, "Subject: [#%d] %s %s is %s\n\n", req.Incident.Id, req.Event.Type, req.Object.Name, req.Event.Severity)
-
 	plugin.FormatMessage(&msg, req)
 
-	err := smtp.SendMail(ch.GetServer(), nil, ch.From, to, bytes.ReplaceAll(msg.Bytes(), []byte("\n"), []byte("\r\n")))
+	return enmime.Builder().
```

Comment:
It seems like `enmime` does not add a Message-ID by default.

Thus, might you please add this header manually. For example like this:
```go
Header("Message-Id", fmt.Sprintf("<%d-%x-%s>", req.Incident.Id, time.Now().UnixNano(), ch.SenderMail)).
```